core/tests/linter_tests/invalid.constants.ajs.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Invalid syntax .ts file, used by scripts/linters/
```

Comment:
Could we please add what is invalid here -- `there should be line break between two constants declaration`?

(Ditto for all invalid files)

Reply:
I think file names are self-explanatory (except one or two files :) -- will fix them in next commit)